src/main/java/net/minecraftforge/network/NetworkInitialization.java

```diff
@@ -5,113 +5,79 @@
 
 package net.minecraftforge.network;
 
+import io.netty.util.AttributeKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraftforge.event.network.CustomPayloadEvent;
+import net.minecraftforge.network.packets.*;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.Marker;
 import org.apache.logging.log4j.MarkerManager;
 import org.jetbrains.annotations.ApiStatus;
 
-import io.netty.util.AttributeKey;
-import net.minecraft.resources.ResourceLocation;
-import net.minecraftforge.network.packets.Acknowledge;
-import net.minecraftforge.network.packets.ChannelVersions;
-import net.minecraftforge.network.packets.LoginWrapper;
-import net.minecraftforge.network.packets.OpenContainer;
-import net.minecraftforge.network.packets.RegistryList;
-import net.minecraftforge.network.packets.RegistryData;
-import net.minecraftforge.network.packets.ConfigData;
-import net.minecraftforge.network.packets.MismatchData;
-import net.minecraftforge.network.packets.ModVersions;
-import net.minecraftforge.network.packets.SpawnEntity;
+import java.util.function.BiConsumer;
 
 @ApiStatus.Internal
 public class NetworkInitialization {
     private static final Logger LOGGER = LogManager.getLogger();
     private static final Marker MARKER = MarkerManager.getMarker("FORGE_NETWORK");
     public static final ResourceLocation LOGIN_NAME = new ResourceLocation("forge", "login");
-    public static final ResourceLocation HANDSHAKE_NAME = new ResourceLocation("forge", "handshake");
-    public static final ResourceLocation PLAY_NAME = new ResourceLocation("forge", "play");
+    private static final ResourceLocation HANDSHAKE_NAME = new ResourceLocation("forge", "handshake");
+    private static final ResourceLocation PLAY_NAME = new ResourceLocation("forge", "network");
     public static final AttributeKey<ForgePacketHandler> CONTEXT = AttributeKey.newInstance(HANDSHAKE_NAME.toString());
 
     public static SimpleChannel LOGIN = ChannelBuilder
```

Comment:
```suggestion
    public static final SimpleChannel LOGIN = ChannelBuilder
```
I know this is inside a class marked as internal, but would be wise to mark this as final to prevent modders from screwing with it anyway. Plus muh static finals perf